alternate-installation-options/index.md

```diff
@@ -8,20 +8,23 @@ layout: default
 {:.cryostat-heading-1}
 Cryostat {{ site.data.versions.cryostat.version }}
 
```

Comment:
Should there be an alternate installation link somewhere else maybe on the masthead? I was confused on where to navigate to this `Other Installation Options` guide since there was only a small link in the `Note` under `Install via OperatorHub`.

Reply:
Not sure. This is supposed to be a more advanced topic, off the mainline "just get this thing running" simple tutorial presented by the `Get Started` page. `Get Started` tries to just choose some defaults at each decision point and send the user down that path. Maybe the masthead should have something like a `Get Started > Tutorial` and `Get Started > Advanced` dropdown, where `Tutorial` is the current `Get Started` and `Advanced` is this doc?

Reply:
Having a dropdown sounds like a much better idea I think.

Reply:
Cool. @cmah88 had also suggested this previously and I filed it as https://github.com/cryostatio/cryostatio.github.io/issues/85 . I still need to figure out how that's actually done... maybe I can bug @insectengine for help.

@maxcao13 are you OK with that update being done separately from this PR and possibly missing the release timing?

Reply:
That's fine, that can be separate and possibly miss the release, maybe I'll look into it as well :-)